docs/user/metal/install_ipi.md

```diff
@@ -177,6 +177,69 @@ pullSecret: ...
 sshKey: ...
 ```
 
+#### Configuration Parameters
+
+| Parameter | Default | Description |
+| --- | --- | --- |
+`libvirtURI` | `qemu://localhost/system` | The location of the hypervisor for running the bootstrap VM. See [Using a Remote Hypervisor](customization_ipi.md#using-a-remote-hypervisor) for more details. |
+`clusterProvisioningIP` | The third address on the provisioning network. `172.22.0.3` | The IP within the cluster where the provisioning services run. |
+`bootstrapProvisioningIP` | The second address on the provisioning network. `172.22.0.2` | The IP on the bootstrap VM where the provisioning services run while the control plane is being deployed. |
+`externalBridge` | `baremetal` | The name of the bridge of the hypervisor attached to the external network. |
+`provisioningBridge` | `provisioning` | The name of the bridge on the hypervisor attached to the provisioning network. |
+`provisioningNetworkInterface` | | The name of the network interface on control plane nodes connected to the provisioning network. |
+`provisioningNetworkCIDR` | `172.22.0.0/24` | The CIDR for the network to use for provisioning. |
+`provisioningDHCPExternal` | | Flag indicating that DHCP for the provisioning network is managed outside of the cluster by existing infrastructure services. |
+`provisioningDHCPRange` | The tenth through the hundredth IP on the provisioning network. `172.22.0.10,172.22.0.100` | The IP range to use for hosts on the provisioning network. |
+`hosts` | | Details about bare metal hosts to use to build the cluster. See [below](#host-properties) for more details. |
+`defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration. |
+`apiVIP` | `api.<clusterdomain>` | The VIP to use for internal API communication. |
+`ingressVIP` | `test.apps.<clusterdomain>` | The VIP to use for ingress traffic. |
+`dnsVIP` | | The VIP to use for internal DNS communication. |
```

Comment:
These 3 either have to be specified, or available in DNS. I could be wrong, the network folks might know

Reply:
I moved them back to this file and added a "VIP Settings" section below.